src/main/java/nextstep/security/authorization/manager/AuthorityAuthorizationManager.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.security.authorization.manager;
+
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+import nextstep.security.authorization.AuthorizationDecision;
+import nextstep.security.authorization.ForbiddenException;
+
+import java.util.Set;
+
+public class AuthorityAuthorizationManager implements AuthorizationManager<HttpServletRequest> {
```

Comment:
```suggestion
public class AuthorityAuthorizationManager implements AuthorizationManager<T> {
```
위에서 나열한 이유에서 사실 `HttpServletRequest`보다는 제네릭 타입을 활용하여 다른 곳에서 활용할 수 있게끔 하는 것이 좋겠죠.

Reply:
필터와 인터셉터 혹은 인터셉터에서도 여러 종류 값을 사용하기 위해서 좋은 선택 같습니다,

그렇다면 AuthorityAuthorizationManager 는 여러 종류의 구현체를 만들어서 관리할 수 도 있을 것 같습니다.

예를 들어 RequestMatcherDelegatingAuthorizationManager 처럼 여러 구현체에  support 메서드나 Matcher를 만듭니다.
이를 통해 처리할 수 있는 인가 작업인지 확인하고 그에 맞는  AuthorityAuthorizationManager  구현체가 작동하는 방식으로 쉽게 관리 될 것 같아요!